tests/test_math/test_compactFock.py

```diff
@@ -83,15 +79,12 @@ def cost_fn():
         assert opt.opt_history[i - 1] >= opt.opt_history[i]
 
 
+@pytest.mark.requires_backend()  # TODO: implement gradient of hermite_renormalized_1leftoverMode
```

Comment:
nice, maybe we can document this somewhere (no args = skip due to interface incompatibility)?